modules/core/shared/src/main/scala/scalacache/AbstractCache.scala

```diff
@@ -3,6 +3,11 @@ package scalacache
 import scala.concurrent.duration.Duration
 
 import scala.language.higherKinds
+import cats.Monad
+import cats.implicits._
```

Comment:
targeted syntax for `functor`, `monadError` and `flatMap`

Reply:
I'm not sure if there is a global standard for whether we want to use fine-grained imports or the whole thing. I can switch to either, but is there any improvement in using the targeted ones? The compilation time is roughly the same, from what I saw last time I checked.

Reply:
I try to use targeted imports as much as possible. Partially for compilation speed, although that is only anecdotal and wouldn't matter anyway on a project of this size. But more importantly for readability, as it gives readers more of a clue where instances and extension methods are coming from.